app/helpers/custom_links_helper.rb

```diff
@@ -0,0 +1,18 @@
+module CustomLinksHelper
+  def custom_links
+    require 'json'
+    result = ''
+    source = CustomSetting['custom_content.header_menus']
+    begin
+      json = JSON.parse(source)
+      json.each do |j|
+        if j.kind_of?(Hash) && j.key?('label') && j.key?('url')
```

Comment:
__[reek]__
* [FeatureEnvy] refers to 'j' more than self (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Feature-Envy.md)

<!-- Comment by [SideCI](https://sideci.com) -->